KubeArmor/core/kubeUpdate.go

```diff
@@ -1696,3 +1767,45 @@ func (dm *KubeArmorDaemon) restoreKubeArmorHostPolicies() {
 		}
 	}
 }
+
+// WatchDefaultPosture Function
+func (dm *KubeArmorDaemon) WatchDefaultPosture() {
```

Comment:
let's make an order like the below to make a group of the functions correlated with each other at the end of this file.

UpdateDefaultPosture()
validateDefaultPosture()
WatchDefaultPosture()